File: Android/sources/com/android/billingclient/api/zzab.java
package com.android.billingclient.api;

import java.util.concurrent.Callable;

public final /* synthetic */ class zzab implements Callable {
    public final /* synthetic */ BillingClientImpl zza;
    public final /* synthetic */ AlternativeBillingOnlyAvailabilityListener zzb;

    public /* synthetic */ zzab(BillingClientImpl billingClientImpl, AlternativeBillingOnlyAvailabilityListener alternativeBillingOnlyAvailabilityListener) {
        this.zza = billingClientImpl;
        this.zzb = alternativeBillingOnlyAvailabilityListener;
    }

    public final Object call() {
        this.zza.zzs(this.zzb);
        return null;
    }
}

File: IOS/v2/com/android/billingclient/api/zzab.swift
package com.iOS.billingclient.api;.swift
import java.util.concurrent.Callable;.swift
public final /* synthetic */ class zzab implements Callable {.swift
public final /* synthetic */ BillingClientImpl zza;.swift
public final /* synthetic */ AlternativeBillingOnlyAvailabilityListener zzb;.swift
public /* synthetic */ zzab(BillingClientImpl billingClientImpl, AlternativeBillingOnlyAvailabilityListener alternativeBillingOnlyAvailabilityListener) {.swift
this.zza = billingClientImpl;.swift
this.zzb = alternativeBillingOnlyAvailabilityListener;.swift
}.swift
public final Object call() {.swift
this.zza.zzs(this.zzb);.swift
return null;.swift
}.swift
}.swift
